### PR TITLE
Fix CSB retries, test pod logging enhancements

### PR DESCRIPTION
**Pull request by smongiar**
- Fix CSB restries
- Store more logs in case of failure

### CODE REVIEW

**Comment by smongiar** [comment]
/pj-rehearse periodic-ci-jboss-fuse-csb-tnb-tests-container-main-csb-ocp4.17-lp-interop-csb-interop-aws-latest

**Review by Croway — APPROVED** [review]

**Comment by smongiar** [comment]
/pj-rehearse periodic-ci-jboss-fuse-csb-tnb-tests-container-main-csb-ocp4.17-lp-interop-csb-interop-aws-latest

**Comment by smongiar** [comment]
/pj-rehearse periodic-ci-jboss-fuse-csb-tnb-tests-container-main-csb-ocp4.17-lp-interop-csb-interop-aws-latest

**Comment by smongiar** [comment]
/pj-rehearse periodic-ci-jboss-fuse-csb-tnb-tests-container-main-csb-ocp4.17-lp-interop-csb-interop-aws-latest

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -359,7 +359,6 @@ aliases:
   - smongiar
   - mcarlett
   - croway
-  - jvrubel
```
Comment (amp-rh) [inline OWNERS_ALIASES:362]: should this change also be mentioned in the commit message?

**Review by amp-rh — APPROVED** [review]

**Review by dfrazzette — APPROVED** [review]

**Pull request merged**